api/filters.go

```diff
@@ -420,6 +420,72 @@ func (api *API) addFilterDimension(w http.ResponseWriter, r *http.Request) {
 	api.respond.JSON(ctx, w, http.StatusCreated, resp)
 }
 
+func (api *API) updateFilterDimension(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	filterID := chi.URLParam(r, "id")
+	dimensionName := chi.URLParam(r, "name")
+
+	logData := log.Data{
+		"filter_id":      filterID,
+		"dimension_name": dimensionName,
+	}
+
+	updatedDimension := model.Dimension{
+		Options: []string{},
```

Comment:
Do you need to initialise this slice here?

Reply:
This was to avoid absent options getting stored/serialised as `null`. Happy to remove if you prefer.